Shared/UI/View Controllers/ViewController.swift

```diff
@@ -32,9 +32,7 @@ class ViewController: UIViewController, Dismissable {
         self.initializeViews()
     }
 
-    func initializeViews() {
-        self.view.translatesAutoresizingMaskIntoConstraints = true
```

Comment:
This fixes some warnings. I don't think should have any negative effects because translatesAutoresizingMaskIntoConstraints is true by default (for programmatic VCs).